### ISSUE TITLE
Merging tickets does not merge the time accounting information

### ISSUE THREAD

**Issue by Suhopoljac** [zammad/zammad#4923]
### Used Zammad Version

6.1. develop

### Environment

- Installation method: any
- Operating system: any
- Database + version:  any
- Elasticsearch version: any
- Browser + version: MS Edge, v. 119.0.2151.44 (Official Build) (64-bit)

### Actual behaviour

Merging tickets results in the time accounting being displayed in two different places, once in the right sidebar in ticket A and once in the right sidebar in ticket B (with which ticket A was merged), instead of displaying the total amount in ticket B. 

before the merge:
f.e. Ticket A: 51 min
Ticket B: 26 min

Total (after merge) 
Ticket A: 51 min
Ticket B: 26 min

### Expected behaviour

After merging the tickets, the total amount of the time billed in ticket A should be added to the amount in ticket B and displayed as a total number. 

before the merge:
f.e. Ticket A: 51 min
Ticket B: 26 min

Total (after merge) in Ticket B: 77 min

### Steps to reproduce the behaviour

1) Activate time accounting
2) Update the tickets with articles, enter a certain number for time accounting purposes. 
3) Merge the ticket A with ticket B.
4) After merging, all articles of the ticket A will be added to the second ticket (B), but the time accounting amount will remain in the original (A) ticket, and the ticket B time accounting amount remains unchanged. 

### Support Ticket

Ticket#10145233

### I'm sure this is a bug and no feature request or a general question.

yes

**Comment by MrGeneration** [comment]
Ticket history and accounted time are not merged.
Up to now this never was a bug but argumented with "working as designed".

**Comment by Suhopoljac** [comment]
I'm open to discussion and may be looking at this from the wrong angle, but since basically all other information gets removed from the original ticket, why are ticket history and accounted time treated differently? Basically, any ticket update/article that is properly listed in the ticket history with timestamps and everything will show up in another ticket.

**Comment by MrGeneration** [comment]
The answer I've received for the past 5 years was 

"technically complicated / too complex"

along side with

"the idea is that you can still see what ticket had what impact"

For this reason the history has been improved at some point to hold the "merged into" information so that you can actually tell exactly what went from where to where.

**Comment by dominikklein** [comment]
I think we have already this one: https://github.com/zammad/zammad/issues/4203

**Comment by cautopartes** [comment]
* I have a ticket with a customer that is using different tickets for time accounting and needs to have the sum of the accounted time in the resulting ticket after merging. 
* I agree that this is mentioned in #4203. Nevertheless, I think having a separate issue for the accounted time that should be merged is justified because of the use case. 
* Remark: Having improved the history of merged tickets, we should also improve in the same way upon the history of split tickets, for the sake of completeness. Currently, a split is not clearly represented in the ticket history.

**Comment by YetAnotherGerrit** [comment]
Isn't the time accounting data stored as an article attribute? I always thought this might be the case, because you can only add time accounting information if you also add an article.

**Comment by MrGeneration** [comment]
No. There's the `time_accounting` attribute in tickets which holds the total and the history contains the sub steps.
